docs/topics/downloader-middleware.rst

```diff
@@ -1015,50 +1015,78 @@ If :attr:`Request.meta <scrapy.http.Request.meta>` has
 the request will be ignored by this middleware even if
 :setting:`ROBOTSTXT_OBEY` is enabled.
 
+Parsers vary in several aspects:
+
+* Language of implementation
+
+* Supported specification
+
+* Support for wildcard matching
+
+* Usage of length based rule: in particular for ``Allow`` and
```

Comment:
:lipstick: I wonder if we should link to some external resource here for users who want to know more about this, e.g. https://developers.google.com/search/reference/robots_txt#order-of-precedence-for-group-member-lines